README.md

```diff
@@ -1,2 +1,114 @@
-# class-data-inheritable
-Class::Data::Inheritable - Inheritable, overridable class data
+NAME
+
+`Class::Data::Inheritable` - Inheritable, overridable class data
+
+SYNOPSIS
+
+    package Stuff;
+    use base qw(Class::Data::Inheritable);
+
+    # Set up DataFile as inheritable class data.
+    Stuff->mk_classdata('DataFile');
+
+    # Declare the location of the data file for this class.
+    Stuff->DataFile('/etc/stuff/data');
+
+    # Or, all in one shot:
+    Stuff->mk_classdata(DataFile => '/etc/stuff/data');
+
+DESCRIPTION
+
+  Class::Data::Inheritable is for creating accessor/mutators to class
+  data. That is, if you want to store something about your class as a
+  whole (instead of about a single object). This data is then inherited by
```

Comment:
This sentence could be a little clearer. I had some issues understanding it until the example was given.

Reply:
Honestly I think most of the text needs a major overhaul. This was just converting it.